style.css

```diff
@@ -117,6 +117,19 @@ h2 {
   margin-top: 1rem;
   margin-bottom: 1rem;
 }
+
+#footer {
```

Comment:
you're adding fixed pixel values to it, which is going to break responsiveness. replace this with responsive units to fix those issues

Reply:
I didnt add those, I think you put that in for the reset search button